### DISCUSSION TITLE
Enhancement: An increasing number of manufacturers are releasing models that generate images, and it is hoped that they can be integrated.

### DISCUSSION THREAD

**Discussion by Passerby1011** [danny-avila/LibreChat#4462] [Feature Requests & Suggestions]
### What features would you like to see added?

More and more manufacturers are launching text-to-image large models, hoping to integrate support for them.

### More details

Many manufacturers use the same interface as OpenAI,
I hope to place all text-to-image large models into a single endpoint. The configuration of the custom endpoint should remain the same as it is now. Each custom endpoint only needs to include text-to-image related configurations and models. LibreChat can integrate the text-to-image large models from multiple custom endpoints into one endpoint in the background.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is planned, there is a lot of discussion on this on our Discord server. The idea is to have Custom Image Endpoints
